README.md

```diff
@@ -0,0 +1,39 @@
+# Ping-Viewer
+
+[![Travis Build Status](https://travis-ci.org/bluerobotics/ping-viewer.svg?branch=master)](https://travis-ci.org/bluerobotics/ping-viewer)
+[![Build status](https://ci.appveyor.com/api/projects/status/y4nl4spd3yw1s4bn/branch/master?svg=true)](https://ci.appveyor.com/project/blueroboticsappveyor/ping-viewer/branch/master)
+[![Discuss](https://img.shields.io/badge/discuss-online-green.svg)](https://discuss.bluerobotics.com/)
+[![Gitter](https://img.shields.io/badge/gitter-online-green.svg)](https://gitter.im/bluerobotics/ardusub/)
+![](https://img.shields.io/github/commit-activity/4w/bluerobotics/ping-viewer.svg)
+![](https://img.shields.io/github/search/bluerobotics/ping-viewer/TODO:.svg)
+
+<a href="https://bluerobotics.com">
+<img src="https://avatars2.githubusercontent.com/u/7120633?v=3&s=200" align="left" hspace="10" vspace="6">
+</a>
+
+
+Graphic user interface for the Ping sonar. Ping is the simple, affordable, and compact ultrasonic altimeter for any aquatic project.
+
+The primary goal of _Ping Viewer_ is to be ease to use for both first time and advanced users. It provides full access to the sensor configuration, setup and visualization via [ping-protocol][1].
+
+All the code is open-source, so you can contribute and evolve it as you want. Follow [Blue Robotics software guidelines][3] for more information.
```

Comment:
```suggestion
All of the code is open-source, so you can contribute and evolve it as you want. Follow [Blue Robotics software guidelines][3] for more information.
```

I think it might be